modules/core/src/main/java/org/apache/ignite/internal/management/snapshot/SnapshotCreateCommand.java

```diff
@@ -33,4 +37,12 @@ public class SnapshotCreateCommand extends AbstractSnapshotCommand<SnapshotCreat
     @Override public Class<SnapshotCreateTask> taskClass() {
         return SnapshotCreateTask.class;
     }
+
+    /** {@inheritDoc} */
+    @Override public String handleException(Exception e, Consumer<String> printer) throws Exception {
+        if (X.hasCause(e, SnapshotWarningException.class))
```

Comment:
Java doc for `SnapshotOperationRequest#warnings`:
> Warnings do not interrupt snapshot process but raise exception at the end to make the operation status 'not OK' if no other error occurred.

We should make this javadoc consistent with current behaviour.
What if one node has warnings and another node has errors? Do we have test for this case?